### ISSUE TITLE
android_integration_test is failing on ci.bazel.io

### ISSUE THREAD

**Issue by meteorcloudy** [bazelbuild/bazel#1591]
```
/private/var/tmp/_bazel_ci/940f5af164c6de2d99dd0946c1f2eaf3/execroot/darwin-x86_64/bazel-out/local-fastbuild/bin/src/test/shell/bazel/android/android_integration_test.runfiles/io_bazel/src/test/shell/bazel/android/android_integration_test: line 170: readelf: command not found
-- Test log: -----------------------------------------------------------
------------------------------------------------------------------------
test_android_binary FAILED: Expected 'libbin', was '' .
/private/var/tmp/_bazel_ci/940f5af164c6de2d99dd0946c1f2eaf3/execroot/darwin-x86_64/bazel-out/local-fastbuild/bin/src/test/shell/bazel/android/android_integration_test.runfiles/io_bazel/src/test/shell/bazel/android/android_integration_test:171: in call to check_soname
/private/var/tmp/_bazel_ci/940f5af164c6de2d99dd0946c1f2eaf3/execroot/darwin-x86_64/bazel-out/local-fastbuild/bin/src/test/shell/bazel/android/android_integration_test.runfiles/io_bazel/src/test/shell/bazel/android/android_integration_test:198: in call to test_android_binary
FAILED: test_android_binary
```

@aj-michael It's probably caused by 49dfc29a40764ae48f2343007f88f3a4938498f1, please take a look. :)

http://ci.bazel.io/job/bazel-tests/BAZEL_VERSION=HEAD,PLATFORM_NAME=darwin-x86_64/160/console

**Comment by meteorcloudy** [comment]
If it can't be fixed in short time, can we rollback this change first?

**Comment by ahumesky** [comment]
It's probably a difference between readelf on linux vs mac

**Comment by damienmg** [comment]
Side-comment: the policy says that the sheriff has all authority for
rolling back and is encouraged to do so if the culprit is identified if the
fix cannot be submitted in the hours.

Another side-comment: this could have been tested pre submit by setting
verified to 1 on the change ;-)

**Comment by meteorcloudy** [comment]
I don't think we should disable android_integration_test on mac, I'll rollback this change.

**Comment by meteorcloudy** [comment]
Fixed by rollback cd7329ac63fd08ba8f914c138117128eb28a5891

**Comment by aj-michael** [comment]
`readelf` is not present on mac.
